### PR TITLE
Removing old implementation of Property Node feature not longer used

### PR DESCRIPTION
**Pull request by cglzaguilar**

### CODE REVIEW

File: source/io/PropertyNode.cpp [hunk 1]
```diff
@@ -1,170 +0,0 @@
...
-
-#include "TypeDefiner.h"
-#include "ExecutionContext.h"
-#include "StringUtilities.h"
-#include "TDCodecVia.h"
-#include "VirtualInstrument.h"
-#include "ControlRef.h"
-#include <stdio.h>
-
-#define VIREO_MODULE_PropertyNode 1
```
[the remark this answers is not in the document]
Comment (cglzaguilar) [reply source/io/PropertyNode.cpp]: Nope, this is code that we dont' have intention on maintaining, we should delete it.

[the remark this answers is not in the document]
Comment (sanmut) [reply source/io/PropertyNode.cpp]: Ok, go for it

**Comment by rajsite** [comment]
Need to also remove https://github.com/ni/VireoSDK/blob/master/source/io/module_propertyNode.js

File: source/io/library_propertyNode.js [hunk 2]
```diff
@@ -1,13 +0,0 @@
-(function () {
-    'use strict';
-    var propertyNodeCAPI = {
-        jsPropertyNodeWrite: function () {
-            Module.propertyNode.jsPropertyNodeWrite.apply(undefined, arguments);
```
Comment (rajsite) [inline source/io/library_propertyNode.js]: Need to delete the corresponding https://github.com/ni/VireoSDK/blob/master/source/io/module_propertyNode.js

Comment (rajsite) [reply source/io/library_propertyNode.js]: And the reference to it in https://github.com/ni/VireoSDK/blob/master/source/core/vireo.loader.shared.js#L6

Comment (cglzaguilar) [reply source/io/library_propertyNode.js]: Done. Thanks for pointing out this file

**Pull request merged**